library/std/src/io/buffered/tests.rs

```diff
@@ -86,6 +87,49 @@ fn test_buffered_reader_seek_relative() {
     assert_eq!(reader.fill_buf().ok(), Some(&[2, 3][..]));
 }
 
+#[test]
+#[unstable(feature = "seek_convenience", issue = "59359")]
```

Comment:
The unstable attribute does not need to be applied to tests. This line can be removed.

Reply:
I think I've seen this somewhere back when writing the tests but don't remember anymore where it was.
Will remove the unstable test attributes.